Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -47,23 +47,23 @@ variable {α β : Type*}
 class SuccOrder (α : Type*) [Preorder α] where
   /-- Successor function -/
   succ : α → α
-  /-- Proof of basic ordering with respect to `succ`-/
+  /-- Proof of basic ordering with respect to `succ`. -/
```

Comment:
Optional: remove the full stop, in all four comments in this file

Reply:
Done!